addons/mail/models/mail_followers.py

```diff
@@ -136,7 +136,7 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None):
 ) AS x
 ORDER BY pid, notif
 """
-            params = [subtype_id, records._name, tuple(records.ids), list(pids) or []]
+            params = [subtype_id, records._name, tuple(records.ids), list(pids or [])]
```

Comment:
Commit:

> [FIX] mail: fix issue when computing _get_recipient_data

I'm surprise you did not include a test case for that.
(Maybe it comes in a following commit?)